maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt

```diff
@@ -349,3 +381,32 @@ data class RenderState(
     val error: String?,
     val downloadUrl: String?,
 )
+
+data class CliVersion(
+    val major: Int,
+    val minor: Int,
+    val patch: Int,
```

Comment:
`patch` is not guaranteed to be an `Int`, i.e. this is possible: `16.0.1-RC`

Reply:
As discussed offline - in this case we will not report any updates available